de-lib/src/main/java/org/iplantc/de/analysis/client/views/dialogs/AnalysisUserSupportDialog.java

```diff
@@ -0,0 +1,237 @@
+package org.iplantc.de.analysis.client.views.dialogs;
+
+
+import org.iplantc.de.theme.base.client.analyses.support.HelpRendererTemplates;
+import org.iplantc.de.client.models.UserInfo;
+import org.iplantc.de.client.models.analysis.AnalysesAutoBeanFactory;
+import org.iplantc.de.client.models.analysis.Analysis;
+import org.iplantc.de.client.models.analysis.AnalysisExecutionStatus;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.event.logical.shared.ValueChangeEvent;
+import com.google.gwt.event.logical.shared.ValueChangeHandler;
+import com.google.gwt.user.client.ui.HTML;
+import com.google.gwt.user.client.ui.Label;
+import com.google.gwt.user.client.ui.TextArea;
+import com.google.inject.Inject;
+
+import com.sencha.gxt.core.client.dom.ScrollSupport;
+import com.sencha.gxt.core.client.util.Margins;
+import com.sencha.gxt.core.client.util.ToggleGroup;
+import com.sencha.gxt.widget.core.client.Window;
+import com.sencha.gxt.widget.core.client.button.TextButton;
+import com.sencha.gxt.widget.core.client.container.VerticalLayoutContainer;
+import com.sencha.gxt.widget.core.client.event.SelectEvent;
+import com.sencha.gxt.widget.core.client.form.CheckBox;
+import com.sencha.gxt.widget.core.client.form.FieldLabel;
+import com.sencha.gxt.widget.core.client.form.FormPanel;
+import com.sencha.gxt.widget.core.client.form.Radio;
+
+import java.util.Date;
+
+
+/**
+ * @author sriram
+ */
+public class AnalysisUserSupportDialog extends Window {
+
+    private TextArea comments;
+    private TextButton submitBtn;
+
+    public interface AnalysisUserSupportAppearance {
+
+        String outputUnexpected();
+
+        String noOutput();
+
+        String backgroudColor();
+
+        String selectCondition();
+
+        String comments();
+
+        String needHelp();
+
+        String submit();
+
+        String agreeToShare();
+
+        String disclaimer();
+
+        String termsOfSupport();
+
+        String supportRequestFailed();
+
+        String supportRequestSuccess();
+
+        String deSystemId();
+    }
+
+
+    private Analysis selectedAnalysis;
+    private HelpRendererTemplates renderer = GWT.create(HelpRendererTemplates.class);
+    private AnalysisUserSupportAppearance appearance = GWT.create(AnalysisUserSupportAppearance.class);
```

Comment:
Ahh! Now I remember why I did not have that injected. I am using appearance in the constructor to initialize button text and background color. If I inject the appearance, it will be `null` because it will be available only after the constructor is complete.